anti-analysis/anti-forensic/hide-shadow-copy.yml

```
@@ -0,0 +1,24 @@
+rule:
+  meta:
+    name: hide shadow copy
```

```suggestion
    name: hide volume shadow copy
```

to make the term slightly more specific. will require renaming the file, too.